evap/staff/importers.py

```diff
@@ -120,6 +132,55 @@ def store_in_database(self, vote_start_datetime, vote_end_date, semester):
             textanswer_visibility=Contribution.TextAnswerVisibility.GENERAL_TEXTANSWERS,
         )
 
+    def check_for_existing_course(self, semester):
```

Comment:
I'd say the name doesn't really describe what the method does, or returns, anymore. Change suggestion a few comments below

Reply:
"below" - doesn't write another comment :microphone: :arrow_down:

Reply:
(below in code. Turns out github keeps the order comments were added at. I wanted to reference https://github.com/e-valuation/EvaP/pull/1683#discussion_r816193814)

Reply:
method is gone anyway :D